code/src/java/pcgen/system/PCGenSettings.java

```diff
@@ -104,6 +106,8 @@ private PCGenSettings()
 		setProperty(VENDOR_DATA_DIR, "@vendordata");
 		setProperty(HOMEBREW_DATA_DIR, "@homebrewdata");
 		setProperty(CUSTOM_DATA_DIR, "@data/customsources".replace('/', File.separatorChar));
+		OutputDB.registerBooleanPreference(
```

Comment:
I feel I'm not properly interpreting your concern here.  This is capturing the preference name (not the value).  (We don't want to capture the value at ANY point because we always want to be able to react to what the setting actually is at that time rather than what it was at PCGen code load)

This registers the (constant) preference name, and that constant is loaded into a BooleanOptionModel.  When FreeMarker is run (and pcgen.prefName is used as an interpolation), then the getAsBoolean() method called, which chains through PropertyContext.getProperty() to determine what the setting actually is.... so the setting is only done "at runtime" - long after both the properties are loaded and OutputDB is initialized.

The use of false is because we have to load the default early on since that is otherwise hardcoded elsewhere that the preference is referred to.

Does that help?